docs/kaneai-release-notes-0-0-2.md

```diff
@@ -0,0 +1,100 @@
+---
+id: kaneai-release-notes-0-0-2
+title: Version 0.0.2 - Beta
+hide_title: false
+sidebar_label: Version 0.0.2
+description: Beta Version 0.0.2
+keywords:
+  - LambdaTest kaneai
+  - LambdaTest kaneai help
+  - LambdaTest kaneai documentation
+  - FAQs
+url: https://www.lambdatest.com/support/docs/kaneai-release-notes-0-0-2/
+site_name: LambdaTest
+slug: kaneai-release-notes-0-0-2/
+---
+import NewTag from '../src/component/newTag';
+import EnhancementTag from '../src/component/enhancementTag';
+import BugFixTag from '../src/component/bugFixTag';
+
+<script type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify({
+       "@context": "https://schema.org",
+        "@type": "BreadcrumbList",
+        "itemListElement": [{
+          "@type": "ListItem",
+          "position": 1,
+          "name": "Home",
+          "item": "https://www.lambdatest.com"
+        },{
+          "@type": "ListItem",
+          "position": 2,
+          "name": "Support",
+          "item": "https://www.lambdatest.com/support/docs/"
+        },{
+          "@type": "ListItem",
+          "position": 3,
+          "name": "Version",
+          "item": "https://www.lambdatest.com/support/docs/kaneai-release-notes-0-0-2/"
+        }]
+      })
+    }}
+></script>
+## 1. Iframe, Shadow DOM, and Canvas Support
+KaneAI now fully supports testing for:
```

Comment:
Club #1 & #2 together and add:

KaneAI now fully supports testing for:
- iframe elements 🖼️
- Shadow DOM elements 🕶️
- Canvas elements 🎨

We've gone the extra mile to bring full iframe, Shadow DOM, and Canvas support in code exports 🖥️. Now, KaneAI handles these intricate UI elements with precision during script generation, making your automation journey even smoother. Whether dealing with complex nested iframes or shadow-rooted elements, KaneAI has you covered!

Reply:
Add tag for web agent